cf/dev.yml

```diff
@@ -0,0 +1,88 @@
+AWSTemplateFormatVersion: '2010-09-09'
+Description: 'cloudtrail-streamer dev template'
+Parameters:
+  LambdaBucket:
+    Description: 'S3 bucket with Lambda code'
+    Type: String
+  LambdaKey:
+    Description: 'S3 bucket key with Lambda code'
+    Type: String
+  KinesisStream:
+    Description: 'Kinesis stream for event submission'
+    Type: String
+Resources:
+  CTStreamerRole:
+    Type: 'AWS::IAM::Role'
+    Properties:
+      AssumeRolePolicyDocument:
+        Statement:
+          - Action:
+              - 'sts:AssumeRole'
+            Effect: Allow
+            Principal:
+              Service: lambda.amazonaws.com
+      Policies:
+        - PolicyDocument:
+            Statement:
+              - Action:
+                  - 's3:GetObject'
+                Resource: !Sub ${S3Bucket.Arn}/*
+                Effect: 'Allow'
+          PolicyName: CTStreamer-GetCloudtrailLogs
+        - PolicyDocument:
+            Statement:
+              - Action:
+                  - 'kinesis:*'
+                Resource: !Sub ${KinesisStream.Arn}
+                Effect: 'Allow'
+          PolicyName: CTStreamer-Kinesis
+        - PolicyDocument:
+            Statement:
+              - Action:
+                  - 'logs:CreateLogGroup'
+                  - 'logs:CreateLogStream'
+                  - 'logs:PutLogEvents'
+                Resource: 'arn:aws:logs:*:*:*'
+                Effect: 'Allow'
+          PolicyName: CTStreamer-Logs
+
+  S3Bucket:
+    Type: 'AWS::S3::Bucket'
+    Properties:
+      BucketName: 'dev-cloudtrail-streamer'
+      NotificationConfiguration:
+        LambdaConfigurations:
+          Function: !Sub ${CTStreamerFunction.Arn}
```

Comment:
Minor, but should we adjust this for the SNS case just for testing purposes in the future if it's intended to be deployed that way in prod?